docker/mlurlphishing/Dockerfile

```diff
@@ -1,4 +1,23 @@
-FROM demisto/python3-deb:3.9.9.25782
+FROM python:3.9.18-slim-bullseye
```

Comment:
should we exclude this script from running to internal auto updates?

Reply:
in case we should you should add this docker-image to be ignored in auto-updates https://github.com/demisto/dockerfiles/blob/master/utils/auto_dockerfile_update/autoupdate-config.json

Reply:
dont think theres a reason to. this is now an external image, since based on a non-demisto image. If theres an update to python/bullsyeye we want to take it